games/cementfactory/index.html

```diff
@@ -5,7 +5,7 @@
 		<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height, target-densitydpi=device-dpi" />
 	</head>
 	<link rel="stylesheet" type="text/css" href="../../css/lcdstyle.css">
-	<script type="text/javascript" src="../../bin/lcdgame.js"></script>
+	<script type="text/javascript" src="../../bin/lcdgame.min.js"></script>
```

Comment:
lcdgame.min.js is built from the source files, not the unminified bundle. As such, the sourcemaps point to the actual files, enabling debugging on those, including breakpoints & co. 

I see no reason to keep the unminified version, so I removed the file.